compiler/gentype/GenTypeConfig.ml

```diff
@@ -105,6 +105,26 @@ let set_debug ~gtconf =
 let compiler_config_file = "rescript.json"
 let legacy_compiler_config_file = "bsconfig.json"
 
+let default_config ~project_root ~bsb_project_root =
+  {
```

Comment:
```ocaml
{ default with project_root; bsb_project_root }
```

Reply:
I made it into a local function instead. with the syntactic improvement.